NOTICE

```diff
@@ -0,0 +1,6 @@
+Apereo uPortal-application-framework aka uw-frame
```

Comment:
I find it odd that there is a `NOTICE` file to say that there are no notices.
But it seems to be an ASF recommendation so :+1: